common/IoT-Introduction.md

```diff
@@ -1,3 +1,101 @@
+
+__Zigbee__  
+The Zigbee network contains three types for basic node types: Zigbee Coordinator (ZC), Zigbee Router (ZR) and Zigbee End Device(ZED). 
+* Coordinator is a special router that forms the network. In a given network there can only be one coordinator. In general, the Hub/Gateway play as coordinator role. 
+* Router has the ability not only to send and receive its packets but also to relay for other nodes in the network. 
+* End Device does not relay packets. An end device relies on its parent for communication with the network. The parent means there is some router node, potentially the coordinator.
+
+![](images/CM-IoT-Introduction/2019-12-20-17-48-03.png)
+
+__Bluetooth Mesh__  
+In Bluetooth Mesh network, the provisioner creates and manages the network by adding, configuring and removing devices into/from the network. The Hub/Gateway can be the provisioner to create the network. A Bluetooth Mesh network consists of different types of nodes: Proxy, Relay, Friend and Low Power node.
+* Relay Node is capable of relaying messages from other nodes and they are essential in increasing the scale and range of a Bluetooth mesh network.
+* Proxy Node can act as a proxy between the Bluetooth mesh network and a Bluetooth device such as a smartphone.
+* Low Power Node can spend most of their lifetime in a low power sleep mode. It needs a friend node that caches any messages targeted to the low power nodes while they are sleeping.
+* Friend Node can cache messages for the low power nodes and store them until low power nodes wake up and fetch the messages. 
+
+# 5. Cloud Service
+From the above description, we know that a Gateway/Hub/Router plays a key role to form and manage the network. It communicates with the local devices with a dedicate protocol like WiFi, Zigbee or Bluetooth Mesh. Also, the Gateway/Hub/Router connects to the Internet through WiFi or Ethernet. It provides a way that your mobile device app can access smart home devices through cloud services. 
+
+The below figure is the Amazon ecosystem smart home model. 
+On the right side of the figure, we can see the Smart Speaker Echo Plus is the Hub that forms the network. It connects with end devices at home via the Zigbee protocol. It builds up a connection with Amazon Alexa Cloud. The mobile device can control end device through the path like this: Alexa Cloud --> Smart Speaker --> End Devices. 
```

Comment:
I think it would be better if modify this sentense: It connects with end devices at home via the Zigbee protocol. It builds up a connection with Amazon Alexa Cloud via Wi-Fi protocol.